packages/components/psammead-most-read/src/Rank/index.jsx

```diff
@@ -82,58 +82,65 @@ const isFiveOrTen = ({ listIndex, service, numberOfItems }) => {
 
 const OneColumnWrapper = styled.div`
   @media (max-width: ${GEL_GROUP_0_SCREEN_WIDTH_MAX}) {
-    min-width: ${props =>
-      listHasDoubleDigits(props.numberOfItems)
-        ? getRankMinWidth(props).group0WithOneColumn
-        : getRankMinWidth(props).group0};
+    flex: 0 0
```

Comment:
I'm not sure if this is the right approach. I worked partly on this and the code is a bit all over the place for Most Read. This is due to having the implementation cater for Most Read pages and also the Most Read component on articles whilst having different sizing for the number typographies. I think what needs to be done is changing the hard coded rank number min widths in [`rankMinWidth.js`](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-most-read/src/utilities/rankMinWidth.js).

As it's Arabic, it's considered a [medium font](https://github.com/bbc/psammead/blob/ff1a3965c1907f3ed1d11a99acc49dec7acaa429/packages/components/psammead-most-read/src/utilities/rankMinWidth.js#L11) size service. So you would want to change one of the column values that have a key of small in [here](https://github.com/bbc/psammead/blob/ff1a3965c1907f3ed1d11a99acc49dec7acaa429/packages/components/psammead-most-read/src/utilities/rankMinWidth.js#L75). Small being the smaller size for articles and default being the size for most read pages. This makes sure the text all aligns squarely.

I can jump on a call if you need any help with this

Reply:
Thank you for the review @HarveyPeachey.

The reason for using `flex-basis` instead of `min-width` is that some browsers have trouble sizing flex items inside flex containers. Also, I have set `flex-grow` and `flex-shrink` to `0` to mimic `min-width`. Doing it this way allows us to specify the initial size of the rank element, before anything else is computed thereby avoiding overlaps.

Reply:
I am happy to jump on a call :)

Reply:
Hmm if the approach works and others are happy then I think it might be fine. My main concern is that changing everything from `min-width` to `flex-basis` would mean we would need to check all 41 services and the variations of most read which include [front page](https://www.bbc.com/arabic), [most read page](https://www.bbc.com/arabic/popular/read) and story pages. Whilst also checking if any browser quirks could be introduced

Reply:
Yeah, it's a bit of an arduous task testing all services and their variations but visually it should remain as it was with `min-width` except in cases where we had overlaps.